### ISSUE TITLE
Accepted scopes need to change

### ISSUE THREAD

**Issue by denny64** [travisghansen/node-red-slack#28]
The latest version of slack no longer has `chat:write:user`, need to change to `chat:write`

https://api.slack.com/scopes/chat:write

**Comment by travisghansen** [comment]
Been a while since I looked at it, does it break the node in some way currently?

**Comment by denny64** [comment]
@travisghansen It gives me an error when I try to send a Slack message. Says acceptedScope is "chat:write:user" and I've only got the scope "chat:write". Where does this check happen? Happy to do a PR if you can point me in the right direction

**Comment by travisghansen** [comment]
@denny64 can you send over the first part of your token (before the `-`)?

My guess is you need to manage this on the slack side and either re-issue a new token or update the one you have (not sure if they even allow that).

**Comment by denny64** [comment]
xoxp

This is the error: http://cl.ly/d9bb857d036c

**Comment by denny64** [comment]
I am using the Bot Token from my custom App. Is that where I should be getting the token?

**Comment by travisghansen** [comment]
Yeah, I created the token I'm using years ago so it may have changed slightly. However, to fully utilize the power of the node you're best off to get yourself an actual `bot` token with starts with `xoxb-`.

When I created mine I used the `hubot` app, but there are many that would probably work.

Pretty sure I did choice 2 (since it was so long ago) from here: https://slack.dev/hubot-slack/#getting-a-slack-token

I haven't fully audited the permissions and it's impossible to do so since different users of the node will invoke different methods. However, the places where the node itself is attempting to do various things behind the scenes I *think* permissions are being checked pre-emptively rather than attempting to invoke and getting a failure. So if you see something off in the logs let me know.

If an actual bot token isn't feasible, working through required scopes to get a functioning node and documenting it is something I'd like to do for sure :)

- https://github.com/yayadrian/node-red-slack/issues/15#issuecomment-471429560
